RELEASE/scripts/autoscend/auto_choice_adv.ash

```diff
@@ -19,8 +19,8 @@ boolean auto_run_choice(int choice, string page)
 		case 20: // See You Next Fall (Itznotyerzitz Mine)
 			itznotyerzitzMineChoiceHandler(choice);
 			break;
-		case 21: // Under the Knife
-			run_choice(2); // skip (The Sleazy Back Alley)
+		case 21: // Under the Knife (The Sleazy Back Alley)

```

Comment:
at some point we should add an option for this for people trying to get the achievement. but this is beyond the scope of this PR...